src/typed-method-types/workflows/triggers/base_response.ts

```diff
@@ -39,33 +46,52 @@ export type BaseTriggerResponse<
           // deno-lint-ignore no-explicit-any
           [otherOptions: string]: any;
         };
+        is_workflow_app: boolean;
         // deno-lint-ignore no-explicit-any
         [otherOptions: string]: any;
       };
+      /** @description A timestamp of when the workflow was created */
+      date_created: number;
       /** @description A timestamp of when the workflow was last updated */
       date_updated: number;
+      /** @description A timestamp of when the workflow was deleted; this property can be 0 when it's not yet deleted. */
+      date_deleted: number;
       // deno-lint-ignore no-explicit-any
       [otherOptions: string]: any;
     };
     /** @description The inputs provided to the workflow */
     inputs: {
+      [key: string]: {
+        value: string;
+        locked: boolean;
+        hidden: boolean;
+      };
+    };
+    outputs: {
```

Comment:
Trigger `outputs` is similar to `workflow.input_parameters` and `workflow.output_parameters` above where details and fields change dependent on the parameter type and instance.

Unless we have a use case where folks are working in this space, I'm inclined to just move to `Record<string, any>` here as well and avoid future maintenance costs